python/ql/test/experimental/dataflow/fieldflow/test.py

```diff
@@ -21,77 +27,135 @@ def SINK_F(x):
         print("OK")
 
 
-# Preamble
+# ------------------------------------------------------------------------------
+# Actual tests
+# ------------------------------------------------------------------------------
+
 class MyObj(object):
     def __init__(self, foo):
         self.foo = foo
 
     def setFoo(self, foo):
         self.foo = foo
 
-
-class NestedObj(object):
-    def __init__(self):
-        self.obj = MyObj("OK")
-
-    def getObj(self):
-        return self.obj
-
-
 def setFoo(obj, x):
     SINK_F(obj.foo)
     obj.foo = x
 
-
-def test_example1():
```

Comment:
The names here used to refer to [the docs](https://github.com/github/codeql/blob/main/docs/ql-libraries/dataflow/dataflow.md#field-flow) (but I can see there were no link in the code), but it might be time to move on from that...

Reply:
Agreed, you have expanded the tests beyond those examples, so telling names are more valuable now.